unconvert.go

```diff
@@ -219,9 +219,6 @@ func main() {
 	}
 
 	importPaths := flag.Args()
```

Comment:
Consider renaming this variable to `patterns`, here, as well as in the functions `mergeEdits` and `computeEdits`. These arguments may be import path patterns, so `patterns` would more accurately describe what is being represented.

It would also match the parameter name of [`packages.Load`](https://godoc.org/golang.org/x/tools/go/packages#Load) where this slice eventually gets passed.

Reply:
You can also add a comment like:

```suggestion
	patterns := flag.Args() // 0 or more import path patterns.
```

That'll make it more clear that it can be an empty slice, and that is okay.

Reply:
Fixed